templates/java/src/main/java/io/appwrite/Client.java.twig

```diff
@@ -33,8 +33,13 @@ public class Client {
         this.endPoint = endPoint;
         this.selfSigned = selfSigned;
         this.headers = new HashMap<>(Map.ofEntries(
-                entry("content-type", "application/json"),
-                entry("x-sdk-version", "{{spec.title | caseDash}}:{{ language.name | caseLower }}:{{ sdk.version }}")
+            entry("content-type", "application/json"),
+            entry("x-sdk-version", "{{spec.title | caseDash}}:{{ language.name | caseLower }}:{{ sdk.version }}") {% if spec.global.defaultHeaders | length > 0 %},{% endif %}
```

Comment:
`entry("x-sdk-version", "{{spec.title | caseDash}}:{{ language.name | caseLower }}:{{ sdk.version }}"){% if spec.global.defaultHeaders | length > 0 %},{% endif %}`

remove the space before the to be added `,` 🙂